src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/api/persist/SQLParsingUtil.java

```diff
@@ -36,15 +41,17 @@
    * @return The record that contain the name string in field
    * @throws IllegalArgumentException throws an exception if we can't find the name in the field
    */
-  private static Record getRecordByName(Result<Record> result, Field<String> field, String name) throws IllegalArgumentException {
+  @VisibleForTesting
+  Record getRecordByName(Result<Record> result, Field<String> field, String name) throws IllegalArgumentException {
```

Comment:
not sure if we really need to change the signature of this function from private to public. I assume we can still call the public API readDataFromSqlResult to test this function ?

Reply:
Changed it back, you're absolutely correct